### ISSUE TITLE
Http provider should call configured endpoint on startup

### ISSUE THREAD

**Issue by Thodor12** [traefik/traefik#9346]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What did you expect to see?

Hello,

I have a tenant like application which serves a website and API across multiple customer configured domains. These domains are stored in my database and are obtained through a small simple internal web application which polls the database.
Traefik uses this little web server to poll for changes in the domain list and then provide the configuration back to Traefik.

However because this contains a database call + some hefty sorting logic in order to properly build the resulting file I don't want to have to call this endpoint all too often (it's not even needed as this doesn't change very much).
So right now I only have it poll **every 5 minutes**.

The problem is that the http provider does not seem to call this endpoint on startup, it first waits 5 minutes before it actually calls the http provider, this would mean that upon a restart of the docker container I'd first have to wait 5 minutes before all of my routes get registered, I could lower the frequency but then I end up in a situation where the endpoint just gets invoked way too much, putting too much strain on my database.

In short, Traefik should, on startup, attempt to poll the http endpoint once before starting the timer until the next run, so that configuration is immediately available at startup and there isn't a delay before it comes available.

**Comment by ldez** [comment]
duplicate of #9110
already fixed by #9116 (part of the release v2.9)
